Header header

float64 minlon
float64 minlat
float64 maxlon
float64 maxlat

uint32[] row_list
uint32[] column_list

uint32 dim
